docs/FORMAT.md

```
@@ -87,10 +87,19 @@ Step-level `runbooks` syntax is shorthand for implicit sequential substeps (`###
 If prompt text appears before a step-level `runbooks` shorthand body, it is attached to the first generated implicit substep only.
 
 where transition is:
-  - { PASS | FAIL | YES | NO } [ { ALL | ANY } ]: result
+  - { PASS | FAIL | YES | NO } [ { ALL | ANY [ AWAIT ] } ]: result
 
 Transitions must appear as list items with the `-` bullet prefix (a dash followed by a space). Paragraph-style transitions (without prefix) are not valid.
 
+### AWAIT modifier
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)